src/control/server/mgmt_system.go

```diff
@@ -613,15 +612,19 @@ func (svc *mgmtSvc) SystemStop(ctx context.Context, pbReq *mgmtpb.SystemStopReq)
 		return nil, errors.New("invalid request, no action specified")
 	}
 
+	// TODO DAOS-7264: system_stop_failed event should be raised in the case
+	//                 that operation failed and should indicate which ranks
+	//                 did not stop.
+	//
 	// Raise event on systemwide shutdown
-	if pbReq.GetHosts() == "" && pbReq.GetRanks() == "" && pbReq.GetKill() {
-		svc.events.Publish(events.New(&events.RASEvent{
-			ID:   events.RASSystemStop,
-			Type: events.RASTypeInfoOnly,
-			Msg:  "System-wide shutdown requested",
-			Rank: uint32(system.NilRank),
-		}))
-	}
+	// if pbReq.GetHosts() == "" && pbReq.GetRanks() == "" && pbReq.GetKill() {
+	// 	svc.events.Publish(events.New(&events.RASEvent{
+	// 		ID:   events.RASSystemStop,
+	// 		Type: events.RASTypeInfoOnly,
+	// 		Msg:  "System-wide shutdown requested",
+	// 		Rank: uint32(system.NilRank),
+	// 	}))
+	// }
```

Comment:
Looks like perhaps this and the system start requested event below were intended to be fully removed?

Reply:
the commented blocks are proceeded with a ticket number, it's a placeholder for modifying the events as the referenced ticket describes, the code had to be commented as this PR changes the event IDs and makes the comment to code invalid, the ticket for updating the system start/stop events is to be done imminently

Reply:
I saw the ticket number, it just looked like the planned new events are totally different from the old ones being removed, so it wasn't clear that the old lines would make a good reference. Not that important in any case.